torch_np/_dtypes_impl.py

```diff
@@ -0,0 +1,74 @@
+"""Dtypes/scalar type implementtaions with torch dtypes.
+
+Here `dtype` is always a torch.dtype, this module knows nothing about
+scalar types, wrapper dtypes or anything like that. PyTorch only.
+"""
+import torch
+
+#### defaults : mimic NumPy, allow user control
+
+
+class DefaultDTypes:
+    """A bag class for dtype defaults."""
+
+    def __init__(self, fp_dtype="numpy", int_dtype="pytorch"):
```

Comment:
Even if it's equivalent, let's set both to `"numpy"`, as in `set_default_dtype`.